pkg/security/probe/selftests/tester.go

```diff
@@ -45,6 +45,7 @@ type SelfTester struct {
 	lastTimestamp   time.Time
 	selfTests       []SelfTest
 	tmpDir          string
+	isClosed        bool
```

Comment:
nit:
isClosed atomic.Bool

Reply:
there is no need for that, all access are under lock